@Test public void testJoin() throws Exception { assertEquals(null, StringUtils.join(null, null, 0, 0)); assertEquals("", StringUtils.join(new Object[0], null, 0, 0)); assertEquals("", StringUtils.join(new Object[0], "", 0, 0)); assertEquals("", StringUtils.join(new Object[0], "", 0, 0)); assertEquals("", StringUtils.join(new Object[0], "", 0, 0)); assertEquals("", StringUtils.join(new Object[0], "", 0, 0)); assertEquals("", StringUtils.join(new Object[0], "", 0, 0)); }